### PR TITLE
My service bugs

### PR DESCRIPTION
**Pull request by sshveta**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 738577f5b47b1df2d91cef62707e06a4913e6e62:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/services/test_myservice.py [hunk 1]
```diff
@@ -154,6 +156,7 @@ def test_retire_service_on_date(myservice):
     myservice.retire_on_date(dt)
 
 
+@pytest.mark.meta(blockers=1219656)
```
Comment (psav) [inline cfme/tests/services/test_myservice.py]: Is a dup of 1218441 - please replace and we'll merge straight away

**Comment by dajoRH** [comment]
Lint report for commit 5097e6ba066b612101f831724f623499d45f6903:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e1bd26109711f636c1ea8e38e2cc81f5a6756fb7:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**